feeder/feeder_config.go

```diff
@@ -112,6 +122,8 @@ func (fs *Setup) Run(in chan observation.InputObservation) error {
 			fs.Feeders[v.Name].SetInputDecoder(format.MakeSuricataInputObservations)
 		case "gamelinux":
 			fs.Feeders[v.Name].SetInputDecoder(format.MakeFjellskaalInputObservations)
+		case "nmsg":
+			fs.Feeders[v.Name].SetInputDecoder(format.MakeNmsgInputObservations)
```

Comment:
Since the `NmsgSocketFeeder` and the `MakeNmsgInputObservations` are intertwined in a way that makes it useless to specify anything but `nmsg` as the format of a `nmsg_socket` feeder (and vice versa), I think it would be nice to do some checking here to warn when encountering any other combination?

Reply:
implemented in 693e550b97e99b3fa114d54790f3f2d9b17d63c2